File: pet-clinic-data/src/main/java/com/sachet/sfgpetclinic/services/VisitService.java
package com.sachet.sfgpetclinic.services;

import com.sachet.sfgpetclinic.model.Visits;

import java.util.Set;

public interface VisitService {

    Set<Visits> findAll();

    Visits findById(Long id);

    Visits save(Visits visits);

    void delete(Visits visits);

    void deleteById(Long id);

}

File: src/main/kotlin/com/sachet/sfgpetclinic/service/VisitService.kt
package com.sachet.sfgpetclinic.service

import com.sachet.sfgpetclinic.model.Visit

interface VisitService {

    fun findById(id: Long): Visit?
    fun findAll(): Set<Visit>
    fun save(visit: Visit): Visit
    fun delete(visit: Visit)
    fun deleteById(id: Long)

}